### PR TITLE
[DYN-39]feat: make homepage for crypta currencies

### PR DESCRIPTION
**Pull request by liubaterehova**
1) I have created hook useSliderPanel - but all content is already was in our project  I just replace it cause I need this hook in 2 components and added check for isCrypto and add second height for our header(HOMEPAGE_HEADER.FULL_CRYPTO_HEIGHT). Cause in CryptoHomepage height of header became bigger due to carousel. And we can't use our header without defining it's height :( 
2) also divided our navigation to different stacks. I think it's become more readable :)
3) a lot of  changes are connected with defining styles

### CODE REVIEW

File: src/screens/More/More.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+import styles from './More.styles';
+
+function More({ navigation }) {
+  return (
+    <View style={styles.screen}>
+      <Card
+        leftIcon={<BitcoinWallet />}
+        rightIcon={<RightArrow />}
+        onCardClick={() => {
```
Comment (apsevkin) [inline src/screens/More/More.jsx:19]: move it out from JSX

**Review by apsevkin — CHANGES_REQUESTED** [review]

**Comment by apsevkin** [comment]
Back arrow on the crypto homepage doesn't work

**Comment by apsevkin** [comment]
When we click more button second time the list with options doesn't show
Screen Shot 2021-03-01 at 17 34 21

File: src/components/HomepageHeader/components/leftIcon/LeftIcon.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+import Text from 'components/Text';
+
+import BackIcon from 'assets/icons/backIcon.svg';
+
+import { getInitials } from 'helpers/utils.helper';
+
+import styles from './LeftIcon.styles';
+
+const AvatarIcon = ({ avatarUrl, username }) => {
```
Comment (arlovaalina) [inline src/components/HomepageHeader/components/leftIcon/LeftIcon.jsx:13]: I think it is better to move AvatarIcon component into separate file

File: src/screens/CryptoHomepage/CryptoHomepage.jsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+import HomepageEmpty from 'components/HomepageEmpty';
+import DailyTransactions from 'components/DailyTransactions';
+import CryptoCarusel from './components/cryptoCarusel';
+
+import styles from './CryptoHomepage.styles';
+
+const getCryptoByIndexCarusel = (index) => {
+  switch (index) {
+    case 0:
+      return { crypto: '₿', text: 'Bitcoin total balance' };
```
Comment (arlovaalina) [inline src/screens/CryptoHomepage/CryptoHomepage.jsx:21]: Maybe let's move these currency signs into file with constants. I saw you use them in many other places.

**Review by arlovaalina — CHANGES_REQUESTED** [review]

**Pull request closed without merging**